[Log{, date=2023-02-07T22:54:19.204510100, level=ERRORname='Lecturelog1', message='The error needs to be resolved, but the program continues to run', stacktrace=null}, Log{, date=2023-02-07T22:54:19.213535400, level=WARNINGname='Lecturelog2', message='Unexpected action, but the system fulfilled the request (catch exception)', stacktrace=null}, Log{, date=2023-02-07T22:54:19.213535400, level=INFOname='Lecturelog3', message='Action that is important to the program (System.out.print)', stacktrace=null}, Log{, date=2023-02-07T22:54:19.213535400, level=DEBUGname='Lecturelog4', message='Expected program action (function result)', stacktrace=null}]